cli/utils.go

```diff
@@ -106,17 +105,21 @@ func loadAlertmanagerConfig(ctx context.Context, alertmanagerURL *url.URL, confi
 		return cfg, nil
 	}
 	if alertmanagerURL != nil {
```

Comment:
not directly related to this PR but if you feel it, this would be a more natural flow:

```go
if alertmanagerURL == nil {
  return nil, errors.New("failed to get Alertmanager configuration")
}
...
```